changelogs/changelog.yaml

```
@@ -1125,3 +1125,266 @@ releases:
+        and will be removed in release 3.0.0 (https://github.com/ansible-collections/community.aws/pull/586).
+      - ec2_elb_info - the boto based ``ec2_elb_info`` module has been deprecated
+        in favour of the boto3 based ``elb_classic_lb_info`` module. The ``ec2_elb_info``
+        module will be removed in release 3.0.0 (https://github.com/ansible-collections/community.aws/pull/586).
+      - elb_classic_lb - the ``elb_classic_lb`` module has been removed and redirected
+        to the ``amazon.aws.ec2_elb_lb`` module which functions identically.
+      - iam - the boto based ``iam`` module has been deprecated in favour of the boto3
+        based ``iam_user``, ``iam_group`` and ``iam_role`` modules. The ``iam`` module
+        will be removed in release 3.0.0 (https://github.com/ansible-collections/community.aws/pull/664).
+      - rds - the boto based ``rds`` module has been deprecated in favour of the boto3
+        based ``rds_instance`` module. The ``rds`` module will be removed in release
+        3.0.0 (https://github.com/ansible-collections/community.aws/pull/663).
+      - script_inventory_ec2 - The ec2.py inventory script is being moved to a new
+        repository. The script can now be downloaded from https://github.com/ansible-community/contrib-scripts/blob/main/inventory/ec2.py
+        and will be removed from this collection in the 3.0 release. We recommend
+        migrating from the script to the `amazon.aws.ec2` inventory plugin.
+      major_changes:
+      - community.aws collection - The community.aws collection has dropped support
+        for ``botocore<1.16.0`` and ``boto3<1.13.0`` (https://github.com/ansible-collections/community.aws/pull/675).
+        Most modules will continue to work with older versions of the AWS SDK, however
+        compatability with older versions of the SDK is not guaranteed and will not
+        be tested. When using older versions of the SDK a warning will be emitted
+        by Ansible (https://github.com/ansible-collections/amazon.aws/pull/442).
+      - community.aws collection - The community.aws collection has dropped support
+        for ``botocore<1.18.0`` and ``boto3<1.15.0`` (https://github.com/ansible-collections/community.aws/pull/711).
+        Most modules will continue to work with older versions of the AWS SDK, however
+        compatability with older versions of the SDK is not guaranteed and will not
+        be tested. When using older versions of the SDK a warning will be emitted
+        by Ansible (https://github.com/ansible-collections/amazon.aws/pull/442).
+      minor_changes:
+      - aws_eks_cluster - Tests for compatability with older versions of the AWS SDKs
+        have been removed (https://github.com/ansible-collections/community.aws/pull/675).
+      - aws_eks_cluster - Tests for compatability with older versions of the AWS SDKs
+        have been removed (https://github.com/ansible-collections/community.aws/pull/675).
```

```suggestion
      - community.aws collection - The community.aws collection has dropped support
        for ``botocore<1.18.0`` and ``boto3<1.15.0`` (https://github.com/ansible-collections/community.aws/pull/711).
        Most modules will continue to work with older versions of the AWS SDK, however
        compatability with older versions of the SDK is not guaranteed and will not
        be tested. When using older versions of the SDK a warning will be emitted
        by Ansible (https://github.com/ansible-collections/amazon.aws/pull/442).
      minor_changes:
      - aws_eks_cluster - Tests for compatability with older versions of the AWS SDKs
        have been removed (https://github.com/ansible-collections/community.aws/pull/675).
```

I suspect this was a quirk of the rebasing?  there's a couple of duplicate entries from 675

---

Probably; cleaned up in the yaml and generated changelog.  Thank you for catching that!